### ISSUE TITLE
Reading code block with empty line returns incorrect format

### ISSUE THREAD

**Issue by ktakashi** [ktakashi/sagittarius-scheme#141]
**[Original report](https://bitbucket.org/ktakashi/sagittarius-scheme/issues/136) by Takashi Kato (Bitbucket: [ktakashi](https://bitbucket.org/ktakashi), GitHub: [ktakashi](https://github.com/ktakashi)).**

----------------------------------------

This should print `pre` but `code`

```
#!scheme
(import (rnrs) (text markdown) (pp))

(pp (string->markdown
          "```
foo

bar
```"))
```

**Comment by ktakashi** [comment]
**Original comment by Takashi Kato (Bitbucket: [ktakashi](https://bitbucket.org/ktakashi), GitHub: [ktakashi](https://github.com/ktakashi)).**

----------------------------------------

Formatting code block

**Comment by ktakashi** [comment]
**Original comment by Takashi Kato (Bitbucket: [ktakashi](https://bitbucket.org/ktakashi), GitHub: [ktakashi](https://github.com/ktakashi)).**

----------------------------------------

Handling empty line in ``` stile verbatim. (Fixes #136)
